new/LIT6605JacobNat.xml

```diff
@@ -48,23 +48,37 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         </profileDesc>
         <revisionDesc>
             <change who="MKr" when="2022-03-26">Created entity</change>
+            <change who="DR" when="2023-05-24">Added bibliography provided by Aaron Butts</change>
```

Comment:
@thea-m you may not merge this very old pull request as in the meanwhile the works starting with 6000 have all been moved to numbered folders, so merging this would create doublets. Please transfer the relevant changes to the master and close this one without merging. Thank you.